StreamR.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+    <AspNetCoreHostingModel>OutOfProcess</AspNetCoreHostingModel>
```

Comment:
This entire attribute should be removed.

Reply:
Yes this was added so I could publish to App Service using AzDO to show preview builds prior to ANCM being updated.